### PR TITLE
Made suggestions

### PR DESCRIPTION
**Pull request by kjgenevieve**
Overall, your code looks great! There were no big issues that I saw. I suggested a few copy edits (spelling), a few formatting edits, and a few ways you could do things a little differently or more efficiently. But as is, the code was readable and clear! You have a successful first app! Congratulations!

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -15,13 +15,15 @@
   <body>
     <h1>Dogs I walk</h1>
     <button id="all">All dogs</button>
-    <h2>All dogs:</h2>
```
Comment (kjgenevieve) [inline index.html:18]: Removed this because it showed up even if you were using the search.

File: index.html [hunk 2]
```diff
@@ -15,13 +15,15 @@
...
     <div id="dogs-container"></div>
     <section>
       <form id="search-form">
         <label for="searchByDogBreed">Search By Dogbreed</label>
-        <input 
-           name="dogBreed"                    id="searchByDogbreed" type="text" placeholder="Enter dogbreed here"
+        <input
```
Comment (kjgenevieve) [inline index.html:22]: Just adjusted the format. :)

File: index.js [hunk 3]
```diff
@@ -1,53 +1,42 @@
 const dogs = []
-document.addEventListener('DOMContentLoaded', () => {
-    fetch('http://localhost:3000/dogs') 
+document.addEventListener("DOMContentLoaded", () => {
```
Comment (kjgenevieve) [inline index.js:2]: You had single quotes here, but double quotes everywhere else.

This is minor; I wouldn't block code from being merged at work for this, but consistency can help a lot when you do a search for something.

File: index.js [hunk 4]
```diff
@@ -1,53 +1,42 @@
...
+    fetch("http://localhost:3000/dogs")
     .then(response => response.json())
     .then(data => getDogs(data))
    
-    const searchForm = document.getElementById("search-form");
+    const searchForm = document.getElementById("search-form")
```
Comment (kjgenevieve) [inline index.js:7]: Similar to the quote comment; you had some semicolons, but not everywhere, so I just removed them. Every company and person has a different preference on them, but the best rule is to be consistent.

Comment (kjgenevieve) [reply index.js:7]: Again, this isn't serious. :)

File: index.js [hunk 5]
```diff
@@ -1,53 +1,42 @@
...
     searchForm.addEventListener("submit", (e) => {searchDogs(e)})    
    
     const allDogsButton = document.getElementById("all")
-    allDogsButton.addEventListener("click", () => {showAllDogs()})
-    
+    allDogsButton.addEventListener("click", () => {
+        const dc = document.getElementById("dogs-container")
```
Comment (kjgenevieve) [inline index.js:12]: I noticed that if I clicked the All Dogs button, it added the dogs. Then if I clicked it again...it added the dogs again. So to prevent having duplicates, I wrote this if statement.

In plain english, if the dogs-container has no children (elements inside of it), add the dogs.

File: index.js [hunk 6]
```diff
@@ -1,53 +1,42 @@
...
+        if (dc.children.length == 0) {
+            showAllDogs()
+        }
+    })
 })
 
-function getDogs(data) {
-    data.forEach((dog) => dogs.push(dog))
-    
+function getDogs(dogData) {
```
Comment (kjgenevieve) [inline index.js:19]: This is also minor! You passed the variable "data" to this function, and I wanted to demonstrate two things:

1. You can change the variable name between functions (if it makes sense to do so).
2. "Data" is vague, especially when you have an app that has multiple fetches, so it's good to be specific.

File: index.js [hunk 7]
```diff
@@ -1,53 +1,42 @@
...
-    findDogs.forEach((dog) => {
-        const h2 = document.createElement("h2");
-        h2.innerText = dog.name;
-        const dogContainer = document.getElementById("searchByDogBreed_container")
-        const img = document.createElement("img")
-        img.src = dog.image
-        img.className = "dog-img"
-        dogContainer.append(h2, img)
-    })
+    e.preventDefault()
```
Comment (kjgenevieve) [inline index.js:24]: This is the big change! It's also okay that you didn't do this.

I noticed that there was a lot of duplicate code in `searchDogs` and `showAllDogs`. They are essentially the same function, just with a different data set and a different place to append the data.

I moved the duplicate code to a new function, `createDogElement`. When you call your `forEach` in either function, it passes each dog to the `createDogElement` one at a time. 

You'll notice that it takes two inputs, too. The second one is the name of the container that the dog needs to be attached to.

File: index.js (hunk 5 above)
Comment (cristinalynn) [reply index.js:12]: I don't see the if statement...

Comment (cristinalynn) [reply index.js:12]: Oh, NVM it's bellow
